doc/tutorials/electrokinetics/electrokinetics.ipynb

```diff
@@ -4,385 +4,708 @@
-    "\\end{equation}\n",
-    "\n",
-    "Here $\\vec{r}$ denotes the spatial coordinate and $t$ the time, while $k$ enumerates the ionic species.\n",
-    "The fluxes are caused by diffusion (due to density variations and external forces) and advection.\n",
-    "\n",
-    "The advective contribution to the flux is given by\n",
-    "\n",
-    "\\begin{equation}\n",
-    "\\vec{j}_{k}^{\\mathrm{adv.}} = c_{k} \\vec{u} ,\n",
-    "\\end{equation}\n",
-    "\n",
-    "where $\\vec{u}(\\vec{r}, t)$ denotes the fluid velocity (advective velocity).\n",
-    "This equation models advection as a simple co-movement of the dissolved ions with the surrounding fluid.\n",
-    "All inertial effects of the ions are neglected.\n",
-    "\n",
-    "The diffusive behavior of the ions is best described in a reference frame co-moving with the local advective velocity $\\vec{u}$.\n",
-    "We assume that the species' relative fluxes instantaneously relax to a local equilibrium.\n",
-    "This assumption allows us to derive the diffusive fluxes from a local free-energy density, which we define as\n",
+    "To use the electrokinetics-algorithm in espresso, one needs to create an instance of the `EKContainer`-object and give it both a `tau` and `solver`, which are the timestep and the poisson-solver, respectively.\n",
+    "Since our species is uncharged, we don't need to solve the electrostatic Poisson-equation, so we can use the placeholder-class, which is called `EKNone`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "eksolver = espressomd.electrokinetics.EKNone(lattice=lattice)\n",
+    "system.ekcontainer = espressomd.electrokinetics.EKContainer(tau=TAU, solver=eksolver)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Adding `EKSpecies` to this container will add them to be integrated by the integrator and optionally be used to calculate the charge-field used to solve the Poisson-equation.\n",
+    "We create an uncharged chemical species and add it to the container to calculate the advection-diffusion. It is initialized with a density of `0.0` and only the diffusion and the advection of the species is enabled. To compare our simulation to the fundamental solution of the Advection-Diffusino equation, we need to approximate a delta-droplet, which can be done by having only a non-zero density at the center of the domain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "species = espressomd.electrokinetics.EKSpecies(\n",
+    "    lattice=lattice, density=0.0, kT=1.0,\n",
+    "    diffusion=DIFFUSION_COEFFICIENT, valency=0.0,\n",
+    "    advection=True, friction_coupling=True,\n",
+    "    ext_efield=[0., 0., 0.], tau=TAU)\n",
+    "system.ekcontainer.add(species)\n",
+    "\n",
+    "species[BOX_L[0] // 2, BOX_L[1] // 2, 0].density = 1.0"
```

Comment:
I would put 1/agrid**3 here to make sure there is one unit of material in the system.
Alternatively would using a gaussian as the starting condition reduce numerical errors connected to discretised gradients?